README.md

```diff
@@ -99,6 +99,15 @@ And see the `X-Envoy-Peer-Metadata-Id` is set to the envoy sidecar id:
 }
 ```
 
+The `X-Envoy-Peer-Metadata-Id` is the Istio HTTP metadata exchange header. These headers will be used for demo purposes, but Istio can be configured to strip the `x-envoy-metadata` headers for requests to service outside the mesh. You can disable this by configuring the meshConfig when installing Istio with:
+```
+  meshConfig:
+    defaultConfig:
+      proxyHeaders:
+        metadataExchangeHeaders:
+          mode: IN_MESH # append istio-meta headers for in-mesh sidecar-to-sidecar requests only
+```
+
 You could also use a DestinationRule to configure TLS origination at the sidecar.
 
```

Comment:
I think it makes more sense to move the diagram before this note to above this note, right?
Because it's a diagram of the above sentence and the below note applies to that case too?

Reply:
I like it right after the metadata example in the response because otherwise it's unclear what it's referring to? Versus `You could also use a DestinationRule to configure TLS origination at the sidecar.` is describing the diagram. But don't feel super strongly.